docs/data/material/guides/interoperability/interoperability.md

```diff
@@ -883,8 +883,3 @@ There is [an ESLint plugin](https://docs.tss-react.dev/detecting-unused-classes)
 ⚠️ **Keep `@emotion/styled` as a dependency of your project**. Even if you never use it explicitly,
 it's a peer dependency of `@mui/material`.
 :::
-
-:::warning
-For [Storybook](https://storybook.js.org): At the time of this writing, Storybook still uses Emotion 10 by default.  
-Material UI and TSS use Emotion 11, so you must make [some changes](https://github.com/InseeFrLab/onyxia-ui/blob/324de62248074582b227e584c53fb2e123f5325f/.storybook/main.js#L31-L32) to `.storybook/main.js` in order to use Storybook with Material UI.
-:::
```

Comment:
I don't think that mentioning Storybook makes any sense in this context, so removed it. Now, we are lucky, the content is outdated anyway: https://twitter.com/storybookjs/status/1578033180883693568.